src/tests/ftest/util/server_utils_params.py

```diff
@@ -453,6 +453,11 @@ def get_params(self, test):
                 self.log.debug("Ignoring the scm_size when scm_class is 'dcpm'")
                 self.scm_size.update(None, "scm_size")
 
+            # Include fault injection settings if configured
+            fault_file = os.environ.get("D_FI_CONFIG")
+            if fault_file and fault_file not in self.env_vars.value:
+                self.env_vars.update("D_FI_CONFIG={}".format(fault_file), "env_vars", True)
```

Comment:
I was thinking about this approach and I was wondering if there could be a situation where `D_FI_CONFIG` may not be cleared between tests.  I'm not sure if we could get into that situation, but since this method is already provided a `TestWithServers` object (through the `test` argument), we have a more accurate means of determining if fault injection is active: `test.fault_injection._fault_file is not None`.  It will probably makes sense to change `FaultInjection._fault_file` to `FaultInjection.fault_file` to allow 'public' access - at least to avoid the pylint error.
```suggestion
            # Include fault injection settings if configured
            if test.fault_injection.fault_file:
                fault_setting = "D_FI_CONFIG={}".format(test.fault_injection.fault_file)
                if fault_setting not in self.env_vars.value:
                    self.env_vars.update(fault_setting , "env_vars", True)
```

Reply:
Sure. Done.